<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWin</class>
 <widget class="QMainWindow" name="ChatWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ChatWin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="msgText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendMsg">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Send msg</string>
    </property>
   </widget>
   <widget class="QListWidget" name="msgsHist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>621</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sendMsg</sender>
   <signal>clicked()</signal>
   <receiver>ChatWin</receiver>
   <slot>sendMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>554</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sendMessage()</slot>
 </slots>
</ui>
